.github/pr-custom-review.yml

```diff
@@ -0,0 +1,8 @@
+rules:
```

Comment:
I was expecting the processbot rules from https://github.com/paritytech/pr-custom-review/issues/32 (equivalent to the [approval rules](https://github.com/paritytech/parity-processbot/tree/4b36d6dcb8dd6d2ba9063c28c1c61bff503c364d#approvals)) to also be included. Doing so allows for deprecating the processbot feature (it is [already planned for that to happen](https://github.com/paritytech/parity-processbot/issues/356#issue-1097813193)).

What's the reason for not including those rules as well?

```yaml
  - name: Required approvals
    check_type: diff
    condition: .*
    any:
      - min_approvals: 2
        teams:
          - core-devs
      - min_approvals: 1
        teams:
          - substrateteamleads
```

Reply:
(Optional) After https://github.com/paritytech/pr-custom-review/pull/57 has landed, you could do it like this:

```diff
  - name: Source files
    check_type: changed_files
-   condition: .*
+   condition:
+     exclude: ^(ci|\.maintain|\.github)\/
    any:
      - min_approvals: 2
        teams:
          - core-devs
      - min_approvals: 1
        teams:
          - substrateteamleads
+ - name: CI files
+   check_type: changed_files
+   condition: ^(ci|\.maintain|\.github)\/
+   min_approvals: 2
```

Reply:
Added rule